Userland/Services/WebContent/WebDriverConnection.cpp

```diff
@@ -1396,6 +1396,47 @@ Messages::WebDriverClient::GetAlertTextResponse WebDriverConnection::get_alert_t
     return JsonValue {};
 }
 
+// 16.4 Send Alert Text, https://w3c.github.io/webdriver/#send-alert-text
+Messages::WebDriverClient::SendAlertTextResponse WebDriverConnection::send_alert_text(JsonValue const& payload)
+{
+    // 1. Let text be the result of getting the property "text" from parameters.
+    // 2. If text is not a String, return error with error code invalid argument.
+    auto text = TRY(get_property(payload, "text"sv));
+
+    // 3. If the current top-level browsing context is no longer open, return error with error code no such window.
+    TRY(ensure_open_top_level_browsing_context());
+
+    // 4. If there is no current user prompt, return error with error code no such alert.
+    if (!m_page_host.has_pending_dialog())
+        return Web::WebDriver::Error::from_code(Web::WebDriver::ErrorCode::NoSuchAlert, "No user dialog is currently open"sv);
+
+    // 5. Run the substeps of the first matching current user prompt:
+    switch (m_page_host.pending_dialog()) {
+    // alert
+    // confirm
```

Comment:
I think we (read: I) mostly use simple ASCII arrows as a replacement for those unselectable fancy arrows in the spec, e.g. here: https://github.com/SerenityOS/serenity/blob/4aeb1ffc1206e5c56e71ad34c8c27cddd9a5f4a5/Userland/Libraries/LibWeb/WebDriver/ExecuteScript.cpp#L81-L97

```suggestion
    // -> alert
    // -> confirm
```